okhttp/src/main/kotlin/okhttp3/HttpUrl.kt

```diff
@@ -1258,7 +1258,7 @@ class HttpUrl internal constructor(
         this.scheme = base.scheme
       } else {
         throw IllegalArgumentException(
-            "Expected URL scheme 'http' or 'https' but no colon was found")
+            "Expected URL scheme 'http' or 'https' but no scheme was found for $input")
```

Comment:
Happy to drop this for PII reasons if there is concern, although as a corrupted url, the debug value seems higher than normal.

Reply:
Log up to 6 characters? Like `www.ex...` ?